docs/refs/api/drivers/DigitalInOut.md

```diff
@@ -2,11 +2,14 @@
 
 Use the DigitalInOut interface as a bidirectional digital pin:
 
-* Read the value of a digital pin when set as an input.
-* Write the value when set as an output.
+* Read the value of a digital pin when set as an `input()`.
+* Write the value when set as an `output()`.
 
 You can use any of the numbered Arm Mbed pins as a DigitalInOut.
 
+**Tips:**
+* Some platforms have a time delay when switching between input and output. 
```

Comment:
what does this mean? Is the time delay specified? is it a wait?

Reply:
on some platoforms you have to insert an arbitrary wait when switching, you cant just use it immediately. This note is here in case they experience trouble and to call attention to it. The example code likewise does this for the same reason.